packages/components/src/styling/src/theming/orbitTheme.ts

```diff
@@ -162,6 +162,8 @@ export interface ColorAliases {
     "text-alias-success": AliasValue;
     "text-alias-static-white": AliasValue;
     "text-alias-input-placeholder": AliasValue;
+    // Focus
+    "focus-ring": AliasValue;
```

Comment:
should the name contains "alias" to match the naming convention of the others?

Reply:
in that case it should be named something like `focus-ring-color-alias-default`